daisy_workflows/export/disk_export.wf.json

```diff
@@ -75,7 +80,9 @@
             "startup-script": "${SOURCE:${NAME}_export_disk.sh}",
             "source-disk-name": "${source_disk}",
             "syft-tar-file": "${SOURCESPATH}/syft.tar.gz",
-            "syft-source": "${syft_source}"
+            "syft-source": "${syft_source}",
+            "sbom-util-source": "${sbom_util_source}",
+            "sbom-util-executable": "${SOURCESPATH}/sbom_util"
```

Comment:
this `sbom-util-executable` derivation is not exactly true. the binary name is `sbomutil` and between the root and the binary there's the release folder. I believe that carrying over only the sbom-util gcs root and then deriving the full path based on that in the function we are downloading it would be better (like we are doing for first party packages). Another option if you want is to change concourse to derive the full path and pass it in, in that case we wouldn't derive it in the daisy workflow - if you want and understand the latter is the best option we can work it out in a way that we can use it even for the first party packages build, changing the approach I introduced.

Reply:
The second approach would be easier for this daisy workflow, because downloading the sbom-util executable directly to the local daisy sources folder will bypass a lot of IAM permission issues.

Reply:
done